src/places.css

```diff
@@ -83,7 +83,7 @@
 }
 
 .ap-input-icon.ap-icon-pin {
-  cursor: initial;
+  cursor: pointer;
```

Comment:
so so related :p

Reply:
(shh - don't look at the name of the branch 🙈)

Reply:
👯